Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
-name = "rustup"
-version = "1.26.0"
-edition = "2021"
+build = "build.rs"
```

Comment:
I'd prefer not to have reorganization of `Cargo.toml` in this PR (for both crates), since this makes it much harder to see actual changes to the contents that are related to the main topic of this PR.

(Unless we move all of the actual `async` changes into one commit that can be reviewed independently, but actually it looks like these `Cargo.toml` changes are a step in the wrong direction IMO.)

Reply:
I can back that out, its a side effect of the vscode even better toml plugin. The actual dep changes of course are needed.